### PR TITLE
Add a new header pointing to existing issues and Discord

### PR DESCRIPTION
**Pull request by WalshyDev**
Mentioned existing issues, mentioned testing on dev if RC and mentioned Discord

### CODE REVIEW

**Comment by WalshyDev** [comment]
oooo, we could even make checkboxes which are like "I have checked for existing issues" and "I am reporting for the latest DEV version"

**Review by TheBusyBiscuit — CHANGES_REQUESTED** [review]
Not a huge fan of the wording on this.
And we already have the discord advertised in the screen before creating the issue.

Also, before we adding to much clutch to this.
I think we should create another feedback form, ask our users for opinion and discuss the structure of this with the @Slimefun/bug-testers team.

**Comment by WalshyDev** [comment]
To the triage team (or anyone else wanting to review), the form of this PR can be found here: https://github.com/Slimefun/Slimefun4/blob/issue-forms-header/.github/ISSUE_TEMPLATE/bug-report-beta.yaml

**Comment by variananora** [comment]
> Not a huge fan of the wording on this.

Yeah, it seems a little off to me.

Also idk if 'related plugins' should have another section to this...
Otherwise it's nicer to have this segmented.

**Comment by brettsaunders21** [comment]
I know I haven't been very active, but I agree with @variananora.

With the old format, it made more sense, but with this one it's a little odd to list related plugins under "Slimefun version".

I think people will just miss out the plugin's part.

File: .github/ISSUE_TEMPLATE/bug-report-beta.yaml [hunk 1]
```diff
@@ -3,6 +3,22 @@ description: Report a Bug or an Issue with Slimefun 4.
 labels: ['🎯 Needs testing', '🐞 Bug Report']
 
 body:
+- type: markdown
+  attributes:
+    value: |
+      ## ❤️ Thank you for wanting to report a bug in Slimefun!
+      If you need any help to identify the problem, visit our [Discord server](https://discord.gg/slimefun)
+      and see if others experience a similar issue.
```
[the remark this answers is not in the document]
Comment (TheBusyBiscuit) [reply .github/ISSUE_TEMPLATE/bug-report-beta.yaml:11]: Yes

**Review by TheBusyBiscuit — DISMISSED** [review]
I made some changes to this.

What do you all think?

**Comment by WalshyDev** [comment]
image

In the checkboxes you forgot to put the stuff under attributes

**Comment by WalshyDev** [comment]
image
as I mentioned before, this is going on a new line, I dislike this... but you said "yes" so err... i guess you can decide to change or not

**Comment by TheBusyBiscuit** [comment]
> image
> as I mentioned before, this is going on a new line, I dislike this... but you said "yes" so err... i guess you can decide to change or not

Fine... I changed it.

**Review by TheBusyBiscuit — APPROVED** [review]
opinions?

**Comment by brettsaunders21** [comment]
I think that looks a lot better. Checkboxes are definitely a good idea.

Bit picky but:
"Videos and Screenshots help illustrate issues the best."

Sounds better in my opinion as:
"The best way to help illustrate the issue is through Videos and Screenshots."

**Review by WalshyDev — COMMENTED** [review]
I'm happy.

As a sidenote, this: 
image

I have sent my support for this to be added

**Comment by TheBusyBiscuit** [comment]
> I'm happy.
> 
> As a sidenote, this:
> image
> 
> I have sent my support for this to be added

In the meantime, they can just be considered single-quotes I guess /shrug

**Comment by TheBusyBiscuit** [comment]
> I think that looks a lot better. Checkboxes are definitely a good idea.
> 
> Bit picky but:
> "Videos and Screenshots help illustrate issues the best."
> 
> Sounds better in my opinion as:
> "The best way to help illustrate the issue is through Videos and Screenshots."

Alright, thanks!

**Review by TheBusyBiscuit — DISMISSED** [review]

**Review by TheBusyBiscuit — APPROVED** [review]

**Review by brettsaunders21 — APPROVED** [review]

**Review by brettsaunders21 — APPROVED** [review]

**Comment by variananora** [comment]
LGTM but should we add another checkbox to make sure their not using modified version of slimefun or something like that?

**Comment by TheBusyBiscuit** [comment]
> LGTM but should we add another checkbox to make sure their not using modified version of slimefun or something like that?

Added.

**Review by TheBusyBiscuit — APPROVED** [review]
I will merge this for now just to see the results but this will still be a beta and I will continue gathering feedback on discord.

**Pull request merged**